Benqi-Agents/Qi-Transfers/jest-when.d.ts

```diff
@@ -0,0 +1 @@
+declare module "jest-when";

```

Comment:
What is this? And why is needed?

Reply:
it was giving me a yellow warning since jest-when has implicitly `any` type although I installed @types/jest-when. so I had to explicitly declare the type. but now I removed it and it doesn't show me the error. I guess it was a glitch in my vscode then.